core-bundle/contao/themes/flexible/styles/main.css

```diff
@@ -1656,20 +1656,24 @@ table.tl_listing {
 	overflow: hidden;
 }
 
+[data-contao--limit-height-target][style*="max-height"] .limit_toggler {
+    background: linear-gradient(transparent, var(--content-bg));
+}
+
 .limit_toggler {
 	width: 100%;
 	position: absolute;
 	bottom: 0;
```

Comment:
@lukasbableck there is one more issue that needs to be fixed

![image](https://github.com/user-attachments/assets/381b5142-e474-4d5f-9e21-6bdf8258fd0c)
The toggler itself needs to be relative to the parent, however you also want it to overlap with the line:

```suggestion
[data-contao--limit-height-target] {
	position: relative;
}

[data-contao--limit-height-target][style*="max-height"] .limit_toggler {
	background: linear-gradient(transparent, var(--content-bg));
}

.limit_toggler {
	width: 100%;
	position: absolute;
	bottom: -1px;
```

Will look like that:

![image](https://github.com/user-attachments/assets/05f2e024-aade-4f17-9d03-51cdae04d2f4)
![image](https://github.com/user-attachments/assets/b379f487-755c-4439-9c4e-02565601359d)